cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/node/CloudNetBridgeModule.java

```diff
@@ -15,14 +15,12 @@
 import de.dytanic.cloudnet.module.NodeCloudNetModule;
 
 import java.io.File;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
 
 public final class CloudNetBridgeModule extends NodeCloudNetModule {
 
     private static final Map<String, String> DEFAULT_MESSAGES = new HashMap<>();
+    private static final ArrayList<String> DEFAULT_HUB_COMMAND_ALIASES = new ArrayList<>();
```

Comment:
Same as above, use Collection instead of an ArrayList.
And you can also use Arrays.asList instead of calling Collection#add multiple times.